### DISCUSSION TITLE
关于cluster模式安全认证问题

### DISCUSSION THREAD

**Discussion by 253980289** [cloudwu/skynet#1979] [General]
当使用skynet的cluster模式时，如果各个cluster节点不在同一物理机上，如何保证双方通讯的安全性，或者说如何做身份认证？

**Comment by cloudwu** [comment]
建议用 vpn 来保证，vpn 原本就是干这个的（而不是翻墙）。

**Comment by 253980289** [comment]
我在想如果能在cluster中增加一些类似hook的的接口是否更方便？比如把cluster里面用socketchannel实现的验证机制通过接口方式开放出来让应用层自己实现具体验证细节。
